example-mojaloop-backend/values.yaml

```diff
@@ -413,10 +261,10 @@ cep-mongodb:
   ## Enable persistence using Persistent Volume Claims
   ## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
   ##
-  persistence:
-    ## @param persistence.enabled Enable MongoDB(&reg;) data persistence using PVC
-    ##
-    enabled: false
```

Comment:
Why was this removed?

Take note that persistence should always be disabled by default, otherwise default installation fail.

Reply:
and again